src/psyclone/tests/test_files/gocean1p0/test00.1_invoke_kernel_using_const_scalar.f90

```diff
@@ -0,0 +1,24 @@
+!-------------------------------------------------------------------------------
```

Comment:
Please change this copyright notice to the one used now (e.g. in `test_files/dynamo0p3/testkern.F90`) and update the date to 2018.